### DISCUSSION TITLE
tts-server --list_models

### DISCUSSION THREAD

**Discussion by charef00** [coqui-ai/TTS#1449] [General Q&A]
i have a problem with windows 10 that say : multiprocessing\context.py", line 192, in get_context
    raise ValueError('cannot find context for %r' % method) from None
ValueError: cannot find context for 'fork'
![Capture](https://user-images.githubusercontent.com/46047976/160235106-6e0a95de-46d7-4058-aad3-1ed36832b89d.PNG)

**Comment by turbulencje** [comment]
If you want to run tts on windows you need to do so in WSL, this bug is marked as wont fix: #867
